python/requirements.txt

```diff
@@ -62,3 +62,4 @@ py-spy>=0.2.0; python_version < '3.12'
 py-spy>=0.4.0; python_version >= '3.12'
 memray; sys_platform != "win32" # memray is not supported on Windows
 pyOpenSSL
+tzlocal
```

Comment:
I'm not sure how to get this available for your tests. When I ssh-ed into ray head, I saw `tzlocal`, so I _think_ it's available normally, but your tests were failing with not being able to find `tzlocal`. I could use some help pointing to how to get the tests happy as I'm otherwise just skimming through the codebase and guessing.